docroot/modules/custom/wateraid_donations/modules/wateraid_donation_stripe/src/Plugin/PaymentProvider/StripePaymentProvider.php

```diff
@@ -44,10 +44,10 @@ public function processWebformComposite(array &$element, FormStateInterface $for
       '#title' => $this->t('Card details'),
     ];
 
-    $submit_button_label = $this->t('Submit');
-    if (!empty($complete_form['elements']['actions']['#submit__label'])) {
-      $submit_button_label = $complete_form['elements']['actions']['#submit__label'];
-    }
+    $submit_button_label = $this->t('Pay now');
```

Comment:
Can we delete code that isn't needed rather than comment it out? Git will allow us to bring it back if it is needed later, and the code will be easier to manage without it.